fivetran_test.go

```diff
@@ -420,3 +420,36 @@ func CreateExternalLogging(t *testing.T) string {
 	}
 	return created.Data.Id
 }
+
+func CreateTempWebhook(t *testing.T) string {
+	t.Helper()
+	webhookId := CreateWebhookAccount(t)
+	t.Cleanup(func() { DeleteWebhook(t, webhookId) })
+	return webhookId
+}
+
+func DeleteWebhook(t *testing.T, id string) {
+	t.Helper()
+	deleted, err := Client.NewWebhookDelete().WebhookId(id).Do(context.Background())
+
+	if err != nil {
+		t.Logf("%+v\n", deleted)
+		t.Error(err)
+	}
+}
+
+func CreateWebhookAccount(t *testing.T) string {
+	t.Helper()
+	created, err := Client.NewWebhookAccountCreate().
+        Url("https://webhook.site/abe96072-249c-40bc-a12d-8b92750175e2").				// Unstable test url
```

Comment:
We don't need working value here. Just any random url that is definitely doesn't exist to avoid unexpected consequences. Something like "http://not-a-real-url/absdefgh123"
Here we are testing API, not webhook itself, so we just need to ensure that passed value is actually set correctly.